packages/gestalt/src/Icon/Icon.js

```diff
@@ -58,6 +60,8 @@ export default function Icon(props: IconProps) {
   );
 }
 
+Icon.icons = IconNames;
```

Comment:
hmmm can we only expose this in dev/testing?
something like: 
```
if (__DEV__) {
  Icon.icons = IconNames
}
```

Reply:
Yeah… Is your concern here bundle size? We’re not currently building multiple envs of Gestalt so adding something like this would be more work than it looks.

I don’t feel strongly about this one - was a request from @christianvuerings to save some typing in the docs.

Reply:
not bundle size, just pointing it out since it does seem like something that is specifically done for testing.  my first thought when i looked at this was "why is this here" and had to go look around for how it was used so, that's why i raised it.

definitely not a concern for this pr.  happy to take this as a follow up action item for myself as well